libs/loggers/source_loggers/video_logger.py

```diff
@@ -93,7 +94,7 @@ def update(self, cv_image, objects, post_processing_data, fps):
         dist_threshold = post_processing_data.get("dist_threshold", 0)
 
         birds_eye_window = np.zeros(self.birds_eye_resolution[::-1] + (3,), dtype="uint8")
-        class_id = int(self.config.get_section_dict('Detector')['ClassID'])
+        class_id = int(self.config.get_section_dict('Detector')['ClassID'])  # TODO: Have a look.
```

Comment:
Have a look regarding what?

Reply:
It was modified, as well as self.image_size in file libs/detectors/detector.py (line 16 and 17), self.model_input_size in libs/classifiers/jetson/face_mask.py (lines: 60-66). We should discuss what to do in endpoint get_processor_info().